client/src/modules/vouchers/toolkit/cash_transfer/cash_transfer.html

```diff
@@ -0,0 +1,91 @@
+<div class="modal-header">
+  <ol class="headercrumb">
+    <li class="static" translate>{{ ToolCtrl.tool.label }}</li>
+    <li class="title" translate>VOUCHERS.GLOBAL.IMPORT_TRANSACTION</li>
+  </ol>
+</div>
+
+<form name="ToolForm" bh-submit="ToolCtrl.import(ToolForm)" bh-form-defaults novalidate>
+  <div class="modal-body">
+
+    <!-- select the cashbox (to debit)  -->
+    <div
+      class="form-group"
+      ng-class="{ 'has-error' : ToolForm.$submitted && ToolForm.cashbox.$invalid }">
+      <label class="control-label" translate>
+        FORM.LABELS.CASHBOX
+      </label> 
+      (<label class="control-label" translate>FORM.LABELS.DEBIT</label>)
+
+      <ui-select
+        name="cashbox"
+        ng-model="ToolCtrl.cashbox"
+        on-select="ToolCtrl.select()"
+        required>
+        <ui-select-match placeholder="{{ 'FORM.PLACEHOLDERS.CASHBOX' | translate }}">
+          <span>{{$select.selected.label}} ({{$select.selected.symbol}})</span>
+        </ui-select-match>
+        <ui-select-choices ui-select-focus-patch repeat="cashbox in ToolCtrl.cashboxes | filter:$select.search">
+          <span ng-bind-html="cashbox.label | highlight:$select.search"></span>
+          (<span ng-bind-html="cashbox.symbol | highlight:$select.search"></span>)
+        </ui-select-choices>
+      </ui-select>
+
+      <div class="help-block" ng-messages="ToolForm.cashbox.$error" ng-show="ToolForm.$submitted">
+        <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
+      </div>
+    </div>
+
+    <!-- select the transfer account (to credit) -->
+    <div 
+      class="form-group"
+      ng-class="{ 'has-error' : ToolForm.$submitted && ToolForm.account.$invalid }">
+      <label class="control-label"  translate>VOUCHERS.GLOBAL.TRANSFER_ACCOUNT</label> 
+      (<label class="control-label"  translate>FORM.LABELS.CREDIT</label>)
+
+      <!-- input typeahead -->
+      <input
```

Comment:
At some point, we should formalize when to use an account typeahead versus an account ui-select.  We could eliminate a lot of code from this PR if we used a component!